<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QMainWindow" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order History</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
    background-color: #F8E7E7;
}
QPushButton {
    background-color: #FFD6D6;
    border: 1px solid #FFAAAA;
    border-radius: 5px;
    font-size: 14px;
}
QTableWidget {
    border: 1px solid #FFAAAA;
    gridline-color: #FFAAAA;
    font-size: 12px;
}
QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333333;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_orders">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>1200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScrollMargin">
     <number>15</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <property name="horizontalHeaderLabels" stdset="0">
     <stringlist>
      <string>Order ID</string>
      <string>Order Status</string>
      <string>Ordered Date</string>
      <string>Shipped Date</string>
      <string>Total</string>
     </stringlist>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="table_order_details">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>1200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <property name="horizontalHeaderLabels" stdset="0">
     <stringlist>
      <string>Product ID</string>
      <string>Product Name</string>
      <string>Unit Price</string>
      <string>Quantity</string>
      <string>Review</string>
     </stringlist>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_orders">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Orders</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_order_details">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Order Details</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
